packages/web-client/app/templates/card-drop/error.hbs

```diff
@@ -0,0 +1,23 @@
+<CardDropPage::Card>
+  <:heading>
+    Whoops!
+  </:heading>
+
+  <:explanation>
+    <p>
+      An error occurred attempting to provision a prepaid card.
+    </p>
+
+    <p data-test-error-message>
+      {{this.message}}
+    </p>
+
+    <Boxel::Button
+      @kind='secondary'
+      @as='anchor'
+      @href={{config 'urls.mailToSupportUrl'}}
+    >
+      Contact Cardstack Support
+    </Boxel::Button>
```

Comment:
This appears together with the 'Visit Cardstack.com' button. Should we only show one CTA?

Reply:
ya makes sense, I’ve added `action` named block to be able to override that default, thanks.